src/openforms/scss/admin/_app_overrides.scss

```diff
@@ -23,3 +23,10 @@
 .signature-image {
   max-width: 350px;
 }
+
+/* Extra login links in admin login screen */
+.admin-login-option {
```

Comment:
Added this because the login link was misaligned (after the text was changed in Open Zaak)
![image](https://user-images.githubusercontent.com/29249171/127015093-8d1021ac-349a-4d99-8cda-2179f246e1b2.png)